### PR TITLE
Temporary fix for many_actor_task.py

### PR DESCRIPTION
**Pull request by williamma12**
## What do these changes do?

Provides a temporary fix for #4255.

### CODE REVIEW

**Comment by williamma12** [comment]
@robertnishihara I tested this with one node and one actor for the sake of time and it works as expected

File: ci/long_running_tests/workloads/many_actor_tasks.py [hunk 1]
```diff
@@ -45,8 +45,10 @@ def method(self):
         self.value += 1
 
 
+# TODO (williamma12): Update the actors to each have only 0.1 of a cpu once
+# https://github.com/ray-project/ray/issues/4312 gets resolved.
 actors = [
-    Actor._remote([], {}, num_cpus=0.1, resources={str(i % num_nodes): 0.1})
+    Actor._remote([], {}, num_cpus=1, resources={str(i % num_nodes): 0.1})
```
Comment (robertnishihara) [inline ci/long_running_tests/workloads/many_actor_tasks.py]: Unfortunately I tried running it and it still doesn't run (I just tried it out).

Let's revert all the changes and just change the remote decorator to have `num_cpus=0.1` and remove the `num_cpus` from the `_remote`

Comment (williamma12) [reply ci/long_running_tests/workloads/many_actor_tasks.py]: Sounds good! I did that and it works as expected now

**Review by robertnishihara — APPROVED** [review]

**Comment by robertnishihara** [comment]
Thanks, this looks great!

**Pull request merged**